File: app/src/main/java/org/jellyfin/androidtv/ui/itemhandling/AudioQueueItem.java
package org.jellyfin.androidtv.ui.itemhandling;

import org.jellyfin.apiclient.model.dto.BaseItemDto;

public class AudioQueueItem extends BaseRowItem {
    public AudioQueueItem(int index, BaseItemDto item) {
        super(index, item);
        this.staticHeight = true;
    }
}

File: app/src/main/java/org/jellyfin/androidtv/ui/itemhandling/AudioQueueItem.kt
package org.jellyfin.androidtv.ui.itemhandling

import org.jellyfin.sdk.model.api.BaseItemDto
import org.jellyfin.sdk.model.api.BaseItemKind

class AudioQueueItem(
	index: Int,
	item: BaseItemDto,
) : BaseRowItem(
	baseRowType = when (item.type) {
		BaseItemKind.PROGRAM -> BaseRowType.LiveTvProgram
		BaseItemKind.RECORDING -> BaseRowType.LiveTvRecording
		else -> BaseRowType.BaseItem
	},
	index = index,
	staticHeight = true,
	baseItem = item,
)
